cl/corpus_importer/management/commands/scrape_pacer_free_opinions.py

```diff
@@ -273,24 +296,52 @@ def get_pdfs(
     courts
     :param index: true if we should index as we process the data or do it later
     :param queue: the queue name
+    :param pdf_days_ago: specify the number of days ago from which to download PDFs
```

Comment:
Sorry, I didn't realize we already had date_start and date_end, but it's a little weird to have two different ways of passing the date range to the function. Why don't we just use these parameters in our cron job and then we don't need pdf_days_ago?

Reply:
you are right, we can just do this:

```shell
manage.py scrape_pacer_free_opinions --action=do-everything --date-start `date -d "2 days ago" +"%Y-%m-%d"` --date-end `date +"%Y-%m-%d"`
```